features/support/minitest_disable.rb

```diff
@@ -0,0 +1,2 @@
+require 'multi_test'
```

Comment:
Ehh, we'll just wait for ruby 3 to turn this on by default, rather than adding:
```ruby
# frozen_string_literal: true
```

to the top of tons of files. too much git churn, imo